checkov/arm/checks/resource/StorageAccountDefaultNetworkAccessDeny.py

```diff
@@ -28,6 +28,8 @@ def scan_resource_conf(self, conf):
         if "properties" in conf:
             if "networkAcls" in conf["properties"]:
                 if "defaultAction" in conf["properties"]["networkAcls"]:
+                    if not isinstance(conf["properties"]["networkAcls"], dict):
```

Comment:
wouldn't it be better to move it before this if condition `if "defaultAction" in conf["properties"]["networkAcls"]:`?

Reply:
It's better! 💯